draft-ietf-lake-edhoc.md

```diff
@@ -310,31 +306,31 @@ An implementation may support only a single method. The Initiator and the Respon
+EDHOC includes the negotiation of connection identifiers (C_I, C_R) which may be used to identify a connection for which EDHOC has established keys. The connection identifiers C_I and C_R do not have any cryptographic purpose in EDHOC.
 
-### Message Correlation {#corr}
+C_I and C_R are chosen by I and R, respectively. Each party chooses a connection identifier it wants the other party to use in its outgoing messages. One byte connection identifiers are realistic in many scenarios as most constrained devices only have a few connections. In cases where a node has only one connection, the identifier may even be the empty byte string.
 
-If the whole transport path provides a mechanism for correlating messages received with messages previously sent, then some of the connection identifiers may be omitted. There are four cases:
+If a connection identifier is used with an application protocol for which EDHOC establishes keys (such as OSCORE) then the connection identifiers SHALL adhere to the requirements for that protocol.  For OSCORE the choice of connection identifier results in the endpoint selecting its Recipient ID, see Section 3.1 of {{RFC8613}}), for which certain uniqueness requirements apply, see Section 3.3 of {{RFC8613}}).
 
-   * corr = 0, the transport does not provide a correlation mechanism.
+Connection identifiers may be used to facilitate the retrieval of protocol state during EDHOC protocol execution. EDHOC transports that do not inherently provide correlation across all messages of an exchange can send connection identifiers along with EDHOC messages to gain that required capability, see {{transport}}). For an example when CoAP is used as transport, see {{coap}}.
 
-   * corr = 1, the transport provides a correlation mechanism that enables the Responder to correlate message_2 and message_1 as well as message_4 and message_3.
+### Transport {#transport}
 
-   * corr = 2, the transport provides a correlation mechanism that enables the Initiator to correlate message_3 and message_2.
+Cryptographically, EDHOC does not put requirements on the lower layers. EDHOC is not bound to a particular transport layer, and can be used in environments without IP. The transport is responsible, where necessary, to handle:
 
-   * corr = 3, the transport provides a correlation mechanism that enables both parties to correlate all three messages.
+* message loss,
+* message reordering,
+* message duplication,
+* correlation of messages, including an indication of a message being message_1,
+* fragmentation, 
+* demultiplex EDHOC messages from other types of messages, and 
+* denial of service protection.
 
-For example, if the key exchange is transported over CoAP, the CoAP Token can be used to correlate messages, see {{coap}}. 
+The Initiator and the Responder need to have agreed on a transport to be used for EDHOC, see {{applicability}}. It is recommended to transport EDHOC in CoAP payloads, see {{coap}}.
```

Comment:
How come it is "recommended" to transport EDHOC in CoAP? The fact that CoAP is the most frequent use case for transporting EDHOC does not make it a "recommended" transport IMO.